seedlet-blocks/theme.json

```diff
@@ -205,8 +206,8 @@
 			},
 			"post-content": {
 				"padding": {
-					"left": "var(--wp--custom--margin--horizontal)",
-					"right": "var(--wp--custom--margin--horizontal)"
+					"left": "20px",
```

Comment:
Just noting that some compiled changes to theme.json are sneaking in here and that's okay.

Reply:
If I just rebuild all children on the trunk I get those changes as well.  IMO they should come in sooner than later, should have already been in with previous changes.  All of the changes look correct, except in that they are unrelated to the change here and should have been made already.